### PR TITLE
Make Server-Timing tests more resilient

### PR DESCRIPTION
**Pull request by cvazac**
Per spec, the performance timeline is not guaranteed to have entries for blocking resources during `load` callbacks - which has led to intermittent failures. Adding `setTimeout` of zero should clean those up.

### CODE REVIEW

File: server-timing/server_timing_header-parsing.https.html [hunk 1]
```diff
@@ -7,7 +7,8 @@
 <head>
     <meta charset='utf-8' />
     <script src="/resources/testharness.js"></script>
-    <script src='/resources/testharnessreport.js'></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/common/performance-timeline-utils.js"></script>
```
Comment (yoavweiss) [inline server-timing/server_timing_header-parsing.https.html]: Including this once seems enough :)

**Review by yoavweiss — APPROVED** [review]
LGTM % nit

File: server-timing/server_timing_header-parsing.https.html (hunk 1 above)
Comment (cvazac) [reply server-timing/server_timing_header-parsing.https.html]: whoops! thx - fixed.

File: common/performance-timeline-utils.js [hunk 2]
```diff
@@ -42,3 +42,10 @@ function test_entries(actualEntries, expectedEntries) {
     }
   })
 }
+
+function delayedLoadListener(callback) {
+  window.addEventListener('load', function() {
+    // TODO(cvazac) Remove this setTimeout when spec enforces sync entries.
```
Comment (gsnedders) [inline common/performance-timeline-utils.js]: Can you file a bug for this to track it?

Comment (cvazac) [reply common/performance-timeline-utils.js]: https://github.com/web-platform-tests/wpt/issues/13874

**Pull request merged**